content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,416 @@
+
+And why not use your DSLR with a macro shooting system to reproduce your silver negatives and use darktable to do this job?
+
+For the purpose of this article, a Lumix Gx8 (20 MP in RAW) with a Leica Elmarit f:2.8 45mm macro was used to digitize silver photos: B&W and color negatives and B&W and color slides. Note that : 
+1. The slides do not pose any problems when processed with darktable (no inversion is required).
+2. B&W negatives are relatively simple to invert and most softwares offer this inversion. In darktable, the classic _invert_ module is sufficient, but negadoctor also manages them.
+3. Color negatives are more capricious because of the basic color of the film (orange) and the white balance. The software that manages them, requires a sample of the unexposed film to recover the RGB values of the orange filter, to subtract them from the exposed negative.
+
+_negadoctor_ allow to process B&W and color negatives. To get the best possible results, it is important to keep a portion of the unexposed film around the image (to take the Dmin, see below) and to follow a rigorous routine that corrects the color in two steps:
+
+1. first the scan is corrected, i.e. the colorimetric deviations brought by the camera or scanner that scanned the film,
+2. the film itself is then corrected, i.e. the colorimetric deviations brought about by the film and by its possible ageing.
+
+The first step is done early in the pipeline, neutralizing the sensor white balance, adjusting the exposure so that the histogram of the image occupies the entire 0-100% range without negative values or clipping, and applying the input color profile (ICC) of the sensor used.
+
+The second step is done in _negadoctor_:
+
+![negadoctor]({filename}en/negadoctor.jpg)
+
+A selector allow to switch to the B&W negative:
+
+![color selector - B&W]({filename}en/negadoctor-select-nb-color.png)
+
+By opening a B&W silver negative:
+
+![Opening - B&W]({filename}en/negadoctor%20N_B%20FR.png)
+
+You only have to click on "black and white" instead of "color" to invert the negative, then to adjust the parameters in the order in which they are in the interface: 
+
+1. For the Dmin, select with the picker an unexposed part of the film. This recovers the minimum density of the film, corresponding to the areas where it is most transparent.
+2. For Dmax, select with the picker a high light area **free of dust or scratches**. This recovers the maximum density of the film, corresponding to the areas where it is the most opaque.
+3. For the scanner exposure bias, select an area that is supposed to be black, or even the whole image. This adjusts the scanner's exposure so that the black areas are black.
```

Comment:
This adjusts the scanner's exposure so that the black areas are black -> This adjusts the scanner's exposure so that the black image areas are matched to values close to black (0, 0, 0).